v2/projects/clipper/index.js

```diff
@@ -11,7 +11,11 @@ module.exports = {
   /* Project Metadata */
   name: "Clipper",
   token: null,
+<<<<<<< HEAD
+  category: "DEX/AMM",
+=======
   category: "DEXes",
+>>>>>>> 76bf44061a96535ca451857abd2418e9131fb4b2
```

Comment:
Thanks for the pull request.
Could you fix this left over conflict here?

Reply:
Ah missed this while merging in. Fixed now, thanks.